src/https_client.c

```diff
@@ -35,6 +36,27 @@ static size_t write_buffer(void *buf, size_t size, size_t nmemb, void *userp) {
   return size * nmemb;
 }
 
+#if defined(IP_TOS)
+static curl_socket_t opensocket_callback(void *clientp, curlsocktype purpose, struct curl_sockaddr *addr)
```

Comment:
Minor style nits:
 * Opening { at end of function definition line.
 * Parentheses around single statement "if (condition) { } else if (condition) { }".

Reply:
Hopefully fixed to your liking